### PR TITLE
BETA: Register m1cro.is-a.dev

### PR DESCRIPTION
**Pull request by m1cro-cat**
Added `m1cro.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/m1cro.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "m1cro-cat",
+        "email": "cybdevteam@hotmail.com"
+    },
+    "record": {
+        "A": ["76.76.21.61"]
```
Comment (DEV-DIBSTER) [inline domains/m1cro.json:7]: We do not support Vercel.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/m1cro.json (hunk 1 above)
Comment (m1cro-cat) [reply domains/m1cro.json:7]: ok, can I have github pages?

Comment (m1cro-cat) [reply domains/m1cro.json:7]: > We do not support Vercel.

I deleted my site from vercel, then I will host on github pages then

Comment (m1cro-cat) [reply domains/m1cro.json:7]: Do I have to do something, or do you yourself? I don't know much about github

Comment (DEV-DIBSTER) [reply domains/m1cro.json:7]: As soon as you get your GitHub page setup, you change it to a CNAME record and have it with your GitHub Domain. Then I'll just approve and you should be good.

Comment (m1cro-cat) [reply domains/m1cro.json]: I hope I did everything right :)

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**